02-alouette/alouette.rb

```diff
@@ -0,0 +1,21 @@
+class Alouette
```

Comment:
I also don't think that we normally provide the rb file and the class and methods. Do we want to adjust them all or remove this one?